test/Microsoft.AspNetCore.SignalR.Tests.Utils/Microsoft.AspNetCore.SignalR.Tests.Utils.csproj

```diff
@@ -5,6 +5,10 @@
     <RootNamespace>Microsoft.AspNetCore.SignalR.Tests</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
+    <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpPackageVersion)" />
```

Comment:
This looks suspicious. Accidentally added somehow?

Reply:
No. TestClient takes dynamic for one of its parameters because the tests are creating hub from type and dynamic allows TestClient to still invoke OnConnected on the hub.

Reply:
Oh yeah, and `dynamic` needs `Microsoft.CSharp`, but it's built in for the `netcoreapp` tfm.